src/dawn/Optimizer/PassStencilSplitter.h

```diff
@@ -31,7 +31,7 @@ class PassStencilSplitter : public Pass {
   /// @brief Maximum number of allowed fields per stencil
   ///
   /// This is the threshold for the splitting pass to be invoked.
-  static constexpr int MaxFieldPerStencil = 40;
+  static constexpr int MaxFieldPerStencil = 2;
```

Comment:
should we revert it? 

Should we control it as we discussed with the use of max-halo? and for the particular stencils where you want to unittest BC?

Reply:
this is still not resolved